docs/ids.md

```diff
@@ -1,27 +1,18 @@
-Often, "ID" is synonymous with the auto-incrementing `id` column in your database of choice. For
-most applications, there really is no distinction. But in event sourcing, you need to think of IDs
-as globally-unique identity—separate from the database that you happen to be using at any given time.
+By default, Verbs uses 64-bit integer IDs called "Snowflakes."
 
-## Globally-Unique Identity?
+## Globally Unique Ids
 
-If you create a new table and insert the first row into it, the `id` value is likely to be `1`. This
-is not globally unique in your system. There are other things with an `id` of `1`.
+We do this because an event-sourcing system needs non-sequential, globablly-unique IDs to run well. Replaying events is a massively powerful feature, but does not pair well with standard auto-incrementing IDs.
+Unique IDs help us both minimize collisions, so that each event is executed with fidelity, and maximize interoperability.
 
-In event sourcing, it's best to use fully unique IDs. This could be UUIDs, which have an astronomically
-low likelihood of collision. The problem with UUIDs is that they're stored as 36-character strings in your
-database, which isn't ideal for indexing.
+We recommend Snowflakes because they are sortable, time-based, and are integers.
+You may also use ULIDs or UUIDs instead; this can be configured in `config/verbs.php`. However, they each introduce some complexity. Both are strings, and UUIDs are not sortable.
 
-While we support UUIDs and ULIDs, we recommend Snowflake IDs. Snowflakes are still unique (although you 
-need to do a little configuration if you're running many servers in parallel), but fit in a 
-`unsigned bigint` column in your database.
+## Snowflakes in Verbs
 
-So instead of a UUID like `4c5433c4-4cfb-4126-81be-44a3af5552a0`, a Snowflake ID will look something
-like `113482333712809984`.
+Verbs uses [`glhd/bits`](https://github.com/glhd/bits) under the hood, and you can use it too. Bits makes it easy to use Snowflakes in Laravel.
```

Comment:
We may specifically want to call out the [Bits configuration](https://github.com/glhd/bits?tab=readme-ov-file#set-the-bits_worker_id-and-bits_datacenter_id) if they're going to run an app on more than one app server.